ext/opcache/zend_accelerator_module.c

```diff
@@ -165,7 +165,12 @@ static ZEND_INI_MH(OnUpdateJit)
 static ZEND_INI_MH(OnUpdateJitDebug)
 {
 	zend_long *p = (zend_long *) ZEND_INI_GET_ADDR();
-	zend_long val = zend_atol(ZSTR_VAL(new_value), ZSTR_LEN(new_value));
+	zend_string *errstr;
+	zend_long val = zend_ini_parse_quantity(new_value, &errstr);
+	if (errstr) {
+		zend_error(E_WARNING, "Invalid \"%s\" setting: %s", ZSTR_VAL(entry->name), ZSTR_VAL(errstr));
+		zend_string_release(errstr);
```

Comment:
Do we need to bailout here because of the warning being emitted here?

Reply:
Maybe in the future, but for now I'm trying to keep this backwards compatible